src/drf_pydantic/parse.py

```diff
@@ -91,7 +94,9 @@ def _convert_field(field: pydantic.fields.ModelField) -> serializers.Field:
     if field.allow_none:
         extra_kwargs["allow_null"] = True
         extra_kwargs["default"] = None
-
+    
+    _check_constraints(field, extra_kwargs)
```

Comment:
I suggest you type what you have in this function same as it's done above. I don't think you need a separate function.

Reply:
I thought it was the best to prevent that function of getting too big but ok

Reply:
It's a purely stylistic request. I think having to jump to another function makes it harder to read code in this particular scenario - especially since equivalent logic is there above and the function called is mutating existing dict.